src/main/java/mate/academy/spring/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -49,7 +50,13 @@ public MovieSessionDaoImpl(SessionFactory sessionFactory) {
     @Override
     public Optional<MovieSession> get(Long id) {
         try (Session session = sessionFactory.openSession()) {
-            return Optional.ofNullable(session.get(MovieSession.class, id));
+            Query<MovieSession> query = session.createQuery(
+                    "FROM MovieSession ms "
+                            + "LEFT JOIN FETCH ms.movie "
+                            + "LEFT JOIN FETCH ms.cinemaHall "
```

Comment:
```suggestion
                            + "JOIN FETCH ms.movie "
                            + "JOIN FETCH ms.cinemaHall "
```
in this case inner join will be enough